bin/build

```diff
@@ -2,7 +2,7 @@
 set -ex
```

Comment:
Is this script needed/used at all anymore? Doesn't look like it to me. Can it be removed? You also made changes here to have it refer to `gofmt`, which seems wrong regardless.

Reply:
No, it's not. Good spot.

Reply:
Looks like this file still hasn't been deleted?